provers/risc0/driver/src/lib.rs

```diff
@@ -59,12 +58,14 @@ impl Prover for Risc0Prover {
         )
         .await;
 
-        let journal: String = result.clone().unwrap().1.journal.encode_hex();
+        let proof: String = result.clone().unwrap().1.journal.encode_hex();
```

Comment:
The journal is PI not the proof, later we can change this to kind of `journal || snark_receipt.snark`, and considering the future zk tier prover, maybe `type || journal || snark_receipt.snark`.